build/pause/Makefile

```diff
@@ -94,6 +95,12 @@ push: .push-$(ARCH)
 	docker push $(IMAGE_WITH_ARCH):$(TAG)
 	touch $@
 
+push-manifest: .manifest-$(ARCH)
+.manifest-$(ARCH): 
```

Comment:
This `.manifest-$(ARCH)` is not really required!

Reply:
.manifest-$(ARCH) not required. target push-manifest is good enough. Thank you. Will update and push.

Reply:
Made a dependency on `.container-$(ARCH)` and pushed the changes.